packages/flutter_tools/test/general.shard/convert_test.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/common.dart';
+import 'package:flutter_tools/src/convert.dart';
+
+import '../src/common.dart';
+
+void main() {
+  late String passedString;
+  late String nonpassString;
+
+  late Utf8Decoder encoder;
+
+  setUp(() {
+    encoder = const Utf8Decoder();
```

Comment:
I'm not an Dart expert, I referred to existing `config_test.dart`. I thought that initialization of variables can be postponed by `setUp` function, missed that it is `const`. If it is needed to be fixed, I will.

Reply:
if it's const, that means there is only a single instance, and it will not change, thus you might as well just initialize it on line 14, as:

```dart
  const Utf8Decoder decoder = Utf8Decoder();
```

Also calling it `decoder` would probably make more sense than `encoder`.

Reply:
Thanks for reviewing my PR. I understand my wrong usage of `const`. Also changed the variable name, too.